src/Template/Users/all.ctp

```diff
@@ -89,22 +89,54 @@ $this->set('title_for_layout', $this->Pages->formatTitle(__('Members')));
             );
             ?>
             </h2>
+
+            <md-menu md-offset="5 50" md-position-mode="target-right target">
+                <md-button ng-click="$mdOpenMenu($event)">
+                    <md-icon>sort</md-icon> Sort by
+                </md-button>
+                <md-menu-content>
+
+                    <?php echo $this->element('sort_option', array(
+                            'param' => 'username',
+                            'direction' => 'asc',
+                            'label' => __('Username (alphabetical)')
+                    ));?>
+
+                    <?php echo $this->element('sort_option', array(
+                            'param' => 'username',
+                            'direction' => 'desc',
+                            'label' => __('Username (reverse alphabetical)')
+                    ));?>
+
+                    <?php echo $this->element('sort_option', array(
+                            'param' => 'since',
+                            'direction' => 'desc',
+                            'label' => __('Newest first')
+                    ));?>
+
+                    <?php echo $this->element('sort_option', array(
+                            'param' => 'since',
+                            'direction' => 'asc',
+                            'label' => __('Oldest first')
+                    ));?>
+
+                    <?php echo $this->element('sort_option_role', array(
```

Comment:
The only difference between `sort_option` and `sort_option_role` is the creation for the pagination link. But `Pagination->sortForRole` is only necessary for displaying the correct arrow. Since your PR gets rid of the arrows, that method isn't useful anymore and you can just use `Paginator->sort()` (i.e. the `sort_option` element) as with the other options (and also remove `Element/sort_options_role.ctp`).

Reply:
As above, I merged in new sort menu element